src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -72,7 +72,7 @@ export const Modal = (props: Props) => {
   };
 
   return (
-    <ContentContainer id="donation-form" data-testid="modal-amount">
+    <ContentContainer>
```

Comment:
why is data-testid removed? is there no testing associated w that?

Reply:
I actually don't know and should have checked, will revert.